pkg/operator/ceph/object/bucket/util.go

```diff
@@ -46,12 +50,24 @@ const (
 	secretSecretKeyKey   = "secretAccessKey"
 	objectStoreName      = "objectStoreName"
 	objectStoreNamespace = "objectStoreNamespace"
+	objectStoreEndpoint  = "endpoint"
 )
 
 func NewBucketController(cfg *rest.Config, p *Provisioner) (*provisioner.Provisioner, error) {
 	const allNamespaces = ""
-	logger.Infof("Ceph Bucket Provisioner launched")
-	return provisioner.NewProvisioner(cfg, provisionerName, p, allNamespaces)
+	provName := provisionerName
+
+	obcWatchOnNamespace, err := k8sutil.GetOperatorSetting(p.context.Clientset, controllerutil.OperatorSettingConfigMapName, "ROOK_OBC_WATCH_ON_OPERATOR_NAMESPACE", "true")
```

Comment:
```suggestion
	obcWatchOnNamespace, err := k8sutil.GetOperatorSetting(p.context.Clientset, controllerutil.OperatorSettingConfigMapName, "ROOK_OBC_WATCH_ON_OPERATOR_NAMESPACE", "false")
```
Shouldn't the default value be `false` for the upgrade case?

Reply:
Yes.